tempesta_fw/http.h

```diff
@@ -139,7 +139,7 @@ typedef struct {
 	int		state;
 	int		_i_st;
 	int		to_read;
-	union {
+	struct {
 		unsigned long acc;
 		unsigned long eol;
 	} _tmp;
```

Comment:
That's all by historical reasons. We had `_tmp_acc` some time ago, then we had `_tmp.{acc,eol}` as a union. Now we have `_tmp.acc` and `_tmp.eol`. Seems that next we'll have is `acc` and `eol`, but not is that patch.